### ISSUE TITLE
Add ChatLoopBackOff to website header

### ISSUE THREAD

**Issue by castrojo** [cncf/cncf.io#880]
We'd like to add a link to CLBO (ChatLoopBackOff) somewhere in here: 

![image](https://github.com/user-attachments/assets/28787a65-e1c0-44d3-b02e-48ac56344169)

Here's a link to the playlist: https://www.youtube.com/playlist?list=PLj6h78yzYM2PnyOsrsCbR_kqjCKfPObHK

cc @k8tgreenley

**Comment by castrojo** [comment]
Would it make sense to integrate it under https://www.cncf.io/online-programs/ perhaps?

**Comment by jeefy** [comment]
@cjyabraham It's a live stream we run that highlights our projects one stream at a time. The streams are hosted by community members, and are meant to be an unboxing style video, where the community member is not necessarily an expert in the project. 

I think there's an argument to put it top-level under "Community Content" but it certainly should fall under the "Online Programs" header.

**Comment by k8tgreenley** [comment]
@cjyabraham I agree it should be under Online Programs. Maybe you and I can jump on a call to discuss the layout of the page and how we integrate in the CLBO playlist

**Comment by cjyabraham** [comment]
Sure. We have a couple of examples of how we pull in the top two videos on a playlist already on the site. See the KCD Highlights section on [the KCD page](https://www.cncf.io/kcds/), for example. Perhaps we could do something similar on the Online Programs page for ChatLoopBackOff?

![Screenshot 2024-08-14 at 7 25 35 PM](https://github.com/user-attachments/assets/e4c12d85-037d-4247-9966-a5bee72d4f7d)

**Comment by k8tgreenley** [comment]
Here is the Google doc: https://docs.google.com/document/d/1rwwwSi48mdbHXSk8LAXXNcmoX6vUCy7OB8kMEuINPbY/edit?usp=sharing

@cjyabraham I like the way we have it set up on the KCD page with a highlight video. Could we do an embedded screen to play the upcoming video live on the webpage?

**Comment by cjyabraham** [comment]
@castrojo I'm a bit confused... is what you're suggesting a brand new page or is it part of the Online Programs page as @k8tgreenley [previously proposed](https://github.com/cncf/cncf.io/issues/880#issuecomment-2287760583)?

@k8tgreenley sure, we can list the videos just like the KCD page.

**Comment by cjyabraham** [comment]
Any updates on this or direction on how to move forward?

**Comment by k8tgreenley** [comment]
I am connect with Riyaz about this and will let you know
